### ISSUE TITLE
[UX] It takes 3 clicks to schedule

### ISSUE THREAD

**Issue by katsar0v** [WordPress/gutenberg#16625]
**Describe the bug**
The bug/issue is that it takes 3 clicks to plan a post. When the time is in the future, the 1st click actually does nothing and just changes the text. The second click schedules the post and the 3rd click confirms it. I think having 2 clicks is more than enough (one would be perfect for me)
 
**To reproduce**
Take a look in the gif
![gutenberg](https://user-images.githubusercontent.com/7236157/61354927-82c27c00-a873-11e9-9ea3-a30c56accfb6.gif)

**Expected behavior**
Having less clicks and more understandable ones would be much better from UX perspective.

**Screenshots**
See attached gif.

**Desktop (please complete the following information):**
 - OS:Ubuntu 18.04
 - Browser Chrome
 - Version `Version 75.0.3770.100 (Official Build) (64-bit)`

**Additional context**
- _Please add the version of Gutenberg you are using in the description._ <- No idea, using WordPress 5.2.2

**Edit**

It seems that the button text changes after the focus from the input is changed. I think the JS event is wrong here.

**Comment by talldan** [comment]
@katsar0v There is an option to remove the 3rd confirmation step. Click the elipsis menu in the top right corner, select Options from the dropdown and then uncheck 'Enable Pre-publish Checks'.

Once I'd done that, I was able to schedule in one click. I didn't have the problem shown in your gif where I had to click an extra time to dismiss the popover:
![schedule](https://user-images.githubusercontent.com/677833/61360058-39445200-a8b0-11e9-9649-24e7508355c1.gif)

Not sure if this could be an OS issue or whether this has been fixed in a recent version of Gutenberg.
